### PR TITLE
New documentation

### PR DESCRIPTION
**Pull request by anirudhpillai**
## What does this PR do?
Created new documentation at /docs.

## ✅ Pull Request checklist

- [ ] Is this code complete?
- [ ] Are tests done/modified?

## 🚨 Is this a breaking change for API clients?
No

## :squirrel: Deploy notes
Update static files

### CODE REVIEW

**Comment by ChristopherHammond13** [comment]
If someone rebuilds statics on this I'll merge it. This is exciting! Will it replace slate?

**Comment by ChristopherHammond13** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.uclapi.com/job/UCLAPI/26/
Jenkins statics successfully built.

**Comment by ChristopherHammond13** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.uclapi.com/job/UCLAPI/27/
Jenkins statics successfully built.

**Comment by ChristopherHammond13** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.uclapi.com/job/UCLAPI/32/
Jenkins statics successfully built.

**Comment by wilhelmklopp** [comment]
@anirudhpillai given the number of  codeclimate issues, which seem to be mostly related to duplicate code, I wonder if there are some more abstractions we can introduce in order to reduce duplicate code. in a follow up PR, perhaps?

File: backend/uclapi/dashboard/tests.py [hunk 1]
```diff
@@ -298,7 +298,7 @@ def test_edit_webhook_POST_ownership_verification_fail(
             content["message"],
             "Ownership of webhook can't be verified."
             "Make sure to follow the documentation: "
-            "https://docs.uclapi.com/#challenge-event"
+            "https://uclapi.com/docs#webhook/challenge-event"
```
Comment (wilhelmklopp) [inline backend/uclapi/dashboard/tests.py]: I think this URL is missing a slash after `docs`

File: backend/uclapi/dashboard/webhook_views.py [hunk 2]
```diff
@@ -104,7 +104,7 @@ def edit_webhook(request):
                 "message": (
                     "Ownership of webhook can't be verified."
                     "Make sure to follow the documentation: "
-                    "https://docs.uclapi.com/#challenge-event"
+                    "https://uclapi.com/docs#webhook/challenge-event"
```
Comment (wilhelmklopp) [inline backend/uclapi/dashboard/webhook_views.py]: same here

File: frontend/src/react/components/app/OAuth.jsx [hunk 3]
```diff
@@ -22,7 +22,7 @@ class OAuth extends React.Component {
         />
         <h4>OAuth Scope</h4>
         <em>
-          <a href="https://docs.uclapi.com/update-me-to-real-value">Scopes documentation</a>
+          <a href="https://uclapi.com/docs#oauth/scopes">Scopes documentation</a>
```
Comment (wilhelmklopp) [inline frontend/src/react/components/app/OAuth.jsx]: same here I believe

File: backend/uclapi/dashboard/tests.py (hunk 1 above)
Comment (anirudhpillai) [reply backend/uclapi/dashboard/tests.py]: The missing slash is intentional as I'm routing to a local element on the page by setting the href to the id of the DOM element. This is how the Material-UI docs work and what I felt was the easiest way to do it.

**Comment by wilhelmklopp** [comment]
Cool! My bad ✨

**Pull request merged**